src/coreclr/jit/instr.cpp

```diff
@@ -1589,7 +1589,6 @@ instruction CodeGen::ins_Copy(var_types dstType)
     {
         return INS_mov;
     }
-<<<<<<< HEAD
 #elif defined(TARGET_WASM)
```

Comment:
Is `instr.cpp` compiled for WASM at all? If not (which I think is the case), we should just delete all `TARGET_WASM`s from it.

Reply:
No, its not used, have removed as suggested